### DISCUSSION TITLE
Aggregated App Events for iOS 14.5+

### DISCUSSION THREAD

**Discussion by pwnywiz** [oddbit/flutter_facebook_app_events#207] [Q&A]
Has anyone managed to configure app events to measure engagement data using aggregated measurement?
The events from iOS 14.5+ are properly logged in Test Events tab, however, when I try to open up the app through the facebook notification (as described here: https://www.facebook.com/business/help/1468510370209420?id=1877298665783613) I cant get facebook to recognise any event and fails.
Is there any additional configuration that anyone has done which allowed them continue with their events setup?

**Comment by pndychrg** [comment]
Hey, can you explain how you made the test events work , as I have registered the app ID and client token too, but unable to get the test events on the meta test events console

**Reply by pwnywiz** [reply]
Hey, im sorry but honestly, i cant remember at all after all this time :(
